environment.py

```diff
@@ -336,7 +336,7 @@ def env() -> Mapping[str, Optional[str]]:
         # choices. For example, the non-retry contribution concurrency
         # determines the number of shards in Elasticsearch.
         #
-        'AZUL_CONTRIBUTION_CONCURRENCY': '64',
+        'AZUL_CONTRIBUTION_CONCURRENCY': '64/8',
```

Comment:
[sn] a lower retry contribution concurrency might'n be necessary (depending on [exp.](https://github.com/DataBiosphere/azul/issues/3629) results)